Resources/Prototypes/Entities/Clothing/OuterClothing/bio.yml

```diff
@@ -9,6 +9,15 @@
     sprite: Clothing/OuterClothing/Bio/general.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Bio/general.rsi
+    clothingVisuals:
+      outerClothing:
+      - state: equipped-OUTERCLOTHING
+      outerClothing-reptilian:
+      - state: equipped-OUTERCLOTHING
+      - state: equipped-OUTERCLOTHING-tail
```

Comment:
The new sprite state should be "-reptilian-tail", to not conflict with future tail-sprites for other species